components/BuildUseCases.vue

```diff
@@ -1,21 +1,19 @@
 <template>
-  <v-stack
-    class="build-usecases"
-    direction="column"
-    sm-direction="column"
-    gap="3.125rem"
-    sm-gap="3.75rem"
-  >
+  <v-stack class="build-usecases" direction="column" sm-direction="column">
     <v-stack
       v-for="useCase in useCasesList"
       :key="useCase.key"
       direction="row"
       sm-direction="column"
-      gap="1.875rem"
-      sm-gap="1.5rem"
-      justify="space-between"
+      gap="10rem"
+      sm-gap="1rem"
```

Comment:
Why did we increase the gap here and remove `justify="space-between"`?

Reply:
It was adding too much space in larger screens, ref: ![Screenshot from 2022-05-25 12-53-42](https://user-images.githubusercontent.com/24249988/170205095-a0f6b4ca-715d-475b-805d-ebdd0b7e975e.png)

Reply:
By adding gap: 10rem the spacing becomes less
![Screenshot from 2022-05-25 12-51-32](https://user-images.githubusercontent.com/24249988/170205399-3ed2cd3c-15d4-48a2-8108-8ad6cdd9c176.png)